openrpc.json

```diff
@@ -1038,6 +1094,29 @@
           }
         }
       },
+      "Caveats": {
+        "description": "Every capability document MAY add restrictions on the way the capability may be used by adding to the caveat property. Capabilities inherit the restrictions from all caveat properties of their parents, and MAY add new caveats in addition to those of their parents.",
+        "type": "array",
+        "items": {
+          "type": "object",
+          "properties": {
+            "type": {
+              "title": "Type",
+              "type": "string",
+              "description": "Type of caveat."
+            },
+            "value": {
+              "title": "CaveatValue",
+              "description": "Value of the caveat."
```

Comment:
this is technically `any`. any thoughts on other ways to represent this @BelfordZ